torchvision/transforms/functional_pil.py

```diff
@@ -280,7 +323,13 @@ def rotate(img, angle, interpolation=0, expand=False, center=None, fill=None):
 
 
 @torch.jit.unused
-def perspective(img, perspective_coeffs, interpolation=Image.BICUBIC, fill=None):
+def perspective(
+    img: Image.Image,
+    perspective_coeffs: float,
+    interpolation=Image.BICUBIC,
```

Comment:
How do we type interpolation :question:

Reply:
As `int`. See https://github.com/python-pillow/Pillow/blob/f518149a1bbe5ebea9d0ca41612e3e6748349ace/src/PIL/Image.py#L184-L190

```suggestion
    interpolation: int = Image.BICUBIC,
```